src/app.css

```
@@ -18,10 +18,19 @@
+}
+
+body {
 	margin: 0;
-	padding: var(--spacing-medium);
+	padding: 0;
+	background: #f0f0f5; /* Fallback color */
+	background: linear-gradient(135deg, #f0f0f5 0%, #d4c8e8 100%);
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	min-height: 100vh;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)